entrypoint.sh

```diff
@@ -100,7 +103,10 @@ fi
 if [ $ignorePolicy ];then
   ARGS="$ARGS --ignore-policy $ignorePolicy"
 fi
+if [ "$hideProgress" == "true" ];then
```

Comment:
You might have to cleanup hideProgress prior to using. See ignoreUnfixed, we have to strip the extra stuff as:
```
ignoreUnfixed=$(echo $ignoreUnfixed | tr -d '\r')
```